### DISCUSSION TITLE
Plug-in for Individual UI Element Resizing

### DISCUSSION THREAD

**Discussion by MaizeMcGee** [runelite/runelite#16779] [Ideas]
A plug-in that would allow you to resize specific elements of the game UI without resizing everything at once like stretched mode.

For instance, you could resize the inventory window to 150% but keep all other UI elements at 100%.

**Comment by pwatts6060** [comment]
I've seen multiple people ask if this is possible in my clan specifically for the chatbox. +1

**Comment by dogoutside** [comment]
this would be huge

**Comment by Hunnersmith** [comment]
They should make it so you can resize the size of your overhead prayer icon as well.
